deps/build.jl

```diff
@@ -1,89 +1,27 @@
-using BinDeps
-using LinearAlgebra, Libdl
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
```

Comment:
If you want to support all Julia versions [`MySQL/deps/build.jl`](https://github.com/JuliaDatabases/MySQL.jl/blob/36eaf2bfbbdd9a27c408d0b2a734fff0d81b63ad/deps/build.jl) is a better example.  In particular, this script should be no-op in Julia v1.3+